packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -12,6 +12,7 @@ import { StyleRules } from '@material-ui/core/styles';
 const styles = (theme: Theme): StyleRules =>
   createStyles({
     bigImage: {
+      alt: 'DataGateway Banner',
```

Comment:
I think we decided not to put alt text on the background image in SciGateway so might be best to remove this from DataGateway too - https://github.com/ral-facilities/scigateway/blob/8403172b97f0ccb98f89bc6b74bac087b4c42216/src/homePage/homePage.component.tsx#L25

Reply:
Done, I have forgotten to remove this